docs/integrations/saas-cloud/box.md

```diff
@@ -80,153 +80,326 @@ _sourceCategory=box  type "event_type" login
-To enable Script Sources you must set the Collector parameter `enableScriptSource` in [user.properties](/docs/send-data/Installed-collectors/collector-installation-reference/user-properties) to true and [restart](/docs/send-data/collection/start-stop-collector-using-scripts.md) the Collector.
-2. Configure the source fields:
-    1. **Name**. (Required) BoxCollector. (Description is optional.)
-    2. **Source Category**. (Required) box
-    3. **Frequency** (Required) **Every 5 Minutes**
-    4. **Specify a timeout for your command**: Active the checkbox and select **60 Minutes**
-    5. **Command** (Required)`/bin/bash` (specify the correct path on your system)
-    6. **Script** (Required) Use the path to `sumojanus`, such as: `/home/ubuntu/sumojanus-box/bin/SumoJanus_Box.bash`. Do not select “Type the script to execute.”
-    7. **Working Directory.** `/home/ubuntu/sumojanus-box`.
-3. Click **Save**.
+### Authentication
+
+You'll need a Box App Key, App Secret, and Access Code to provide to Sumo Logic.
+
+The following steps must be followed to get the credentials:
+1. Login into the [Box Account](https://app.box.com/login).
+2. Create and register a new app from the [App Console](https://app.box.com/developers/console). To register the App with Box follow [these](https://developer.box.com/guides/authentication/jwt/jwt-setup/#app-creation-steps) steps. Select **Server Authentication (with JWT) **as the authentication method. Note that use of a key pair requires  2-step verification to be enabled on Box.
+3. Generate `public private key pair` as described in the following steps [Key Pair](https://developer.box.com/guides/authentication/jwt/jwt-setup/#public-and-private-key-pair) and download the JSON file.
+4. Go to the `Configuration` and change `App Access Level` to `App + Enterprise Access` and enable `Manage Enterprise properties` in `Application Scopes` and save changes as shown below.<br/><img src={useBaseUrl('img/send-data/box-source4.png')} alt="Box" /> <br/><br/><img src={useBaseUrl('img/send-data/box-source5.png')} alt="Box" /> <br/><img src={useBaseUrl('img/send-data/box-source6.png')} alt="Box" />
+5. Authorize your app by following these steps [Authorize](https://developer.box.com/guides/authentication/jwt/jwt-setup/#app-authorization).
+
+
+### States
+
+A Box Source tracks errors, reports its health, and start-up progress. You’re informed, in real-time, if the Source is having trouble connecting, if there's an error requiring user action, or if it is healthy and collecting by utilizing [Health Events](/docs/manage/Health-Events).
+
+A Box Source goes through the following states when created:
+1. **Pending**: Once the Source is submitted it is validated, stored, and placed in a **Pending** state.
+2. **Started**: A collection task is created on the Hosted Collector.
+3. **Initialized**: The task configuration is complete in Sumo Logic.
+4. **Authenticated**: The Source successfully authenticated with Box.
```

Comment:
**Authenticated**. The Source successfully authenticated with Box.

Reply:
done